test/event_test.py

```diff
@@ -28,6 +28,70 @@
 )
 
 
+names_and_events = (
+    ('NoEvent', pygame.NOEVENT),
+    ('ActiveEvent', pygame.ACTIVEEVENT),
+    ('KeyDown', pygame.KEYDOWN),
+    ('KeyUp', pygame.KEYUP),
+    ('MouseMotion', pygame.MOUSEMOTION),
+    ('MouseButtonDown', pygame.MOUSEBUTTONDOWN),
+    ('MouseButtonUp', pygame.MOUSEBUTTONUP),
+    ('JoyAxisMotion', pygame.JOYAXISMOTION),
+    ('JoyBallMotion', pygame.JOYBALLMOTION),
+    ('JoyHatMotion', pygame.JOYHATMOTION),
+    ('JoyButtonDown', pygame.JOYBUTTONDOWN),
+    ('JoyButtonUp', pygame.JOYBUTTONUP),
+    ('VideoResize', pygame.VIDEORESIZE),
+    ('VideoExpose', pygame.VIDEOEXPOSE),
+    ('Quit', pygame.QUIT),
+    ('SysWMEvent', pygame.SYSWMEVENT),
+
+    ('UserEvent', pygame.USEREVENT),
+    ('UserEvent', pygame.USEREVENT + 1),
+    ('UserEvent', pygame.NUMEVENTS - 1),
+
+    ('Unknown', 0xFFFF),
+)
+
+# Add in any SDL 2 specific events.
+if pygame.get_sdl_version()[0] >= 2:
+        names_and_events += (
+            ('FingerMotion', pygame.FINGERMOTION),
+            ('FingerDown', pygame.FINGERDOWN),
+            ('FingerUp', pygame.FINGERUP),
+            ('MultiGesture', pygame.MULTIGESTURE),
+
+            # These can be corrected when issue #1221 is resolved.
+            # Should be: 'AudioDeviceAdded'
+            ('Unknown', pygame.AUDIODEVICEADDED),
+            # Should be: 'AudioDeviceRemoved'
+            ('Unknown', pygame.AUDIODEVICEREMOVED),
+
+            ('MouseWheel', pygame.MOUSEWHEEL),
+            ('TextInput', pygame.TEXTINPUT),
+            ('TextEditing', pygame.TEXTEDITING),
+            ('WindowEvent', pygame.WINDOWEVENT),
+
+            ('ControllerAxisMotion', pygame.CONTROLLERAXISMOTION),
+            ('ControllerButtonDown', pygame.CONTROLLERBUTTONDOWN),
+            ('ControllerButtonUp', pygame.CONTROLLERBUTTONUP),
+            ('ControllerDeviceAdded', pygame.CONTROLLERDEVICEADDED),
+            ('ControllerDeviceRemoved', pygame.CONTROLLERDEVICEREMOVED),
+            ('ControllerDeviceMapped', pygame.CONTROLLERDEVICEREMAPPED),
+
+            ('DropFile', pygame.DROPFILE),
+        )
+
+        # Add in any SDL 2.0.5 specific events.
+        if pygame.get_sdl_version() >= (2, 0, 5):
+            names_and_events += (
+                # These can be corrected when issue #1223 is resolved.
```

Comment:
This should be part of the PR, and the PR made to encompase all issues, isntead of just marking them as "Unknown", when they clearly _aren't_.

Reply:
The missing drop and audio events are related to ifdef/ifndef statements which are not being address by this PR. I think it is best to leave them as 'Unknown' until they are properly fixed.